lib/domino.dart

```diff
@@ -1,5 +1,9 @@
 import 'dart:async';
 
+import 'src/element/setter.dart';
```

Comment:
the `Setter` interface itself could be in `domino.dart`, and no need for importing and exporting.

Reply:
Fixed it